site/layouts/alert/single.html

```diff
@@ -62,6 +62,16 @@ <h1 class="text--white">{{ .Title }}</h1>
               </td>
               <td>{{ .Params.wasc }}</td>
             </tr>
+            <tr> 
+              <td>
+                <strong>Tags</strong>
+              </td>
+              <td>
+              {{ range $tagkey, $tagvalue := .Params.alerttags }}
+              	<a href="{{ $tagvalue }}">{{ upper $tagkey }}</a><br>
```

Comment:
This is okay for now but we should note that not all tags will have links as values...

Reply:
Yes, its assuming all of the ones we define will be links. If that changes then this will need to change too :)